fast64_internal/f3d/flipbook.py

```diff
@@ -12,26 +13,27 @@ class TextureFlipbook:
     name: str
     exportMode: str
     textureNames: list[str]
+    images: list[tuple[bpy.types.Image, FImage]]
```

Comment:
This change was made without updating a bunch of `TextureFlipbook` instanciations such as
`TextureFlipbook(arrayName, "Array", textureList)`
which breaks a bunch of code